components/benefit_sponsors/app/models/benefit_sponsors/benefit_packages/benefit_package.rb

```diff
@@ -232,9 +232,8 @@ def renew_member_benefit(census_employee)
 
         # family.validate_member_eligibility_policy
         if true #family.is_valid?
-          enrollments = family.enrollments.by_benefit_sponsorship(benefit_sponsorship)
-          .by_effective_period(predecessor_application.effective_period)
-          .enrolled_and_waived
+          enrollments = family.active_household.hbx_enrollments.enrolled_and_waived
```

Comment:
@nks2109 @raghuramg any reason we have `if true` here on line 234?

@raghuramg can you see if ticket 38639 is still valid?

Reply:
@nks2109 @raghuramg see above